LoginApp(new)/LoginWindow/mainwindow.cpp

```diff
@@ -0,0 +1,96 @@
+#include "mainwindow.h"
+#include "ui_mainwindow.h"
+#include <QMessageBox>
+#include <QPixmap>
+#include <qresource.h>
+MainWindow::MainWindow(QWidget *parent) :
+    QMainWindow(parent),
+    ui(new Ui::MainWindow)
+{
+    ui->setupUi(this);
+    SetWindow();
+    connect(ui->pushBtn_login, &QPushButton::clicked, this, &MainWindow::onPushbtnLoginClick);
+    connect(ui->lineEdit_username, &QLineEdit::textChanged, this, &MainWindow::UsernameChanged);
+    connect(ui->lineEdit_password, &QLineEdit::textChanged, this, &MainWindow::UsernameChanged);
+    //connect(...) connect button startTest test with browser call;
+}
+
+void MainWindow::UsernameChanged()
+{
+    bool isLoginActivated = (ui->lineEdit_username->text() != "") && (ui->lineEdit_password->text() != "");
+    ui->pushBtn_login->setEnabled(isLoginActivated);
+}
+
+void MainWindow::RunCamera()
+{
+    QScopedPointer<CameraWidget> camera_widget(new CameraWidget());
+    camera_widget->exec();
+    if(camera_widget->is_photo_saved_)
+    {
+         ui->pushButton_start_test->setVisible(true);
+         ui->pushBtn_login->setVisible(false);
+         ui->lineEdit_username->setReadOnly(true);
+         ui->lineEdit_password->setReadOnly(true);
+         ui->pushButton_start_test->setVisible(true);
+    }
+
+}
+
+void MainWindow::SetWindow()
+{
+    this->setWindowTitle("Login");
+    this->setMinimumSize(470, 400);
+    this->setMaximumSize(470, 400);
+    QPixmap logo(":/img/Images/share-image.png");
+    int width = ui->label_SoftServeLogo->width();
+    int height = ui->label_SoftServeLogo->height();
+    ui->label_SoftServeLogo->setPixmap(logo.scaled(width, height, Qt::KeepAspectRatio));
+    ui->pushBtn_login->setEnabled(false);
+    ui->pushButton_start_test->setVisible(false);
+}
+
+bool MainWindow::IsUsernameAndPasswordValid()
+{
+    QString username = ui->lineEdit_username->text();
+    QString password = ui->lineEdit_password->text();
+    if(username == "12345" && password == "12345")
+        return true;
+    return false;
+}
+
+void MainWindow::AskCameraAccess()
+{
+    QString message = tr("You have successfully completed login. \nTo start test take your photo.");
+    QMessageBox msgBox(QMessageBox::Information, tr("Confirm login"), message);
+    QIcon title(":/icons/Icons/Mario.png");
+    msgBox.setWindowIcon(title);
+    QPushButton *acceptBtn = msgBox.addButton(tr("&Turn on camera") ,QMessageBox::ActionRole);
+    QPushButton *abortBtn = msgBox.addButton(QMessageBox::Abort);
+    msgBox.exec();

```

Comment:
You don't need to check clicked button. See: http://doc.qt.io/qt-5/qmessagebox.html#exec